File: baekjoon/p12780/Main.java
package baekjoon.p12780;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.next(), sc.next()));
    }

    private static int solve(String n, String h) {
        int ret = 0;
        int idx = -1;

        while (true) {
            idx = n.indexOf(h, idx+1);
            if(idx == -1) {
                break;
            }

            ret++;
        }

        return ret;
    }

}

File: baekjoon/p12780/Main.kt
package baekjoon.p12780

fun main() {
    println(solve(readLine()!!, readLine()!!))
}

fun solve(h: String, n: String): Int {
    var ret = 0
    var idx = -1

    while (true) {
        idx = h.indexOf(n, idx + 1)
        if (idx == -1) {
            break
        }

        ret++
    }

    return ret
}